pom.xml

```diff
@@ -48,20 +53,26 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
+                    <compilerArgs>
+                        <arg>--add-exports</arg>
+                        <arg>
+                            java.desktop/sun.swing=ALL-UNNAMED,java.security.jgss/sun.security.jgss=ALL-UNNAMED,java.base/sun.net.spi=ALL-UNNAMED,java.desktop/sun.awt=ALL-UNNAMED,java.base/sun.security.x509=ALL-UNNAMED
+                        </arg>
+                    </compilerArgs>
                     <compilerId>groovy-eclipse-compiler</compilerId>
                 </configuration>
                 <dependencies>
                     <dependency>
                         <groupId>org.codehaus.groovy</groupId>
                         <artifactId>groovy-eclipse-compiler</artifactId>
-                        <version>3.4.0-sb-fixed</version>
+                        <version>3.5.0-sb-fixed</version>
                     </dependency>
                     <dependency>
                         <groupId>org.codehaus.groovy</groupId>
                         <artifactId>groovy-eclipse-batch</artifactId>
-                        <version>2.4.3-01</version>
+                        <version>2.4.17-01</version>
```

Comment:
The current version we use is 2.4.17-02.

Reply:
I've picked this version from RA pom

Reply:
https://github.com/SmartBear/axm/blob/5c37d4d1eb794eb33c0966629c361cfc82d51953/pom.xml#L46